sampleApp/sample.tsx

```diff
@@ -19,6 +20,15 @@ const render = (globalProps: GlobalProps, notebooks: Notebook[]) => {
 	);
 };
 
+const renderDropdown = (globalProps: GlobalProps, notebooks: Notebook[], label: string, popupVisible: boolean) => {
+	ReactDOM.render(
+		<OneNotePickerDropdown globals={globalProps.globals} notebooks={notebooks} dropdownLabel={label} popupDirection={'bottom'} popupVisible={popupVisible}/>,
+		document.getElementById('oneNotePickerDropdown') as HTMLElement
```

Comment:
Regarding the label, we can break it down into a couple of sets of values:
1) The name of the selection (if selectedId exists)
2) Some default label. In most cases, this might say something like 'Default Location' or 'Select location (optional)' (if selectedId does not exist).
We can get this information from the props. You could do something like "if selectedId exists, find the name of the item and display it. If it does not exist, fall back to some default string". This way you can avoid having that label parameter in the renderDropdown() function